sdk/keyvault/keyvault-keys/src/index.ts

```diff
@@ -717,6 +757,63 @@ export class KeyClient {
       return { value: result.value! };
     });
   }
+
+  /**
+   * Gets the rotation policy of a Key Vault Key.
+   *
+   * Example usage:
+   * ```ts
+   * let client = new KeyClient(vaultUrl, credentials);
+   * await client.setKeyRotationPolicy("MyKey", myPolicy);
+   * let result = await client.getKeyRotationPolicy("myKey");
+   * ```
+   *
+   * @param name - The name of the key.
+   * @param options - The optional parameters.
+   */
+  public getKeyRotationPolicy(
+    name: string,
+    options: GetRotationPolicyOptions = {}
+  ): Promise<KeyRotationPolicy | undefined> {
+    return withTrace("getRotationPolicy", options, async () => {
+      const policy = await this.client.getKeyRotationPolicy(this.vaultUrl, name);
+      if (policy.id) {
```

Comment:
I did ask the service team about what's expected here. Right now, this will return 

```
{ id: undefined, lifetimeActions: [], attributes: null }
```

And I wonder if it should 404 or 204 with no content. Depending on what I hear back we can decide what the client library should do. But for now, I made the decision to return undefined in that case.